Assets/Prefabs/Player/Player.prefab

```diff
@@ -0,0 +1,188 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &6984755896152334966
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 6984755896152334962}
+  - component: {fileID: 6984755896152334963}
+  - component: {fileID: 6984755896152334957}
+  - component: {fileID: 2341849012200123104}
+  - component: {fileID: 6984755896152334956}
+  - component: {fileID: 6984755896152334959}
+  - component: {fileID: 6443449700755736254}
+  m_Layer: 0
```

Comment:
Also, I had created a layer for 'Player' so once you merge in my changes, just need to set this player prefab to the 'Player' layer

Reply:
Player prefab is now set to the "Player" layer